src/render_tasks/d3d12_raytracing_task.hpp

```diff
@@ -154,7 +161,21 @@ namespace wr
 			auto& data = fg.GetData<RaytracingData>(handle);
 			auto& as_build_data = fg.GetPredecessorData<wr::ASBuildData>();
 
-			d3d12::DescriptorHeap* desc_heap = cmd_list->m_rt_descriptor_heap->GetHeap();
+			d3d12::DescriptorHeap* heap = cmd_list->m_rt_descriptor_heap->GetHeap();
+
+			if (d3d12::GetRaytracingType(device) == RaytracingType::FALLBACK)
```

Comment:
This is duplicate in all 3 raytracing tasks. This can be abstracted into a function.